layouts/partials/analytics/hotjar.html

```diff
@@ -0,0 +1,10 @@
+<script>
+    (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:'{{ .Site.Params.matomoAnalytics.hjid }}',hjsv:6};
```

Comment:
Did you forget to add the new param or is this the wrong config key?

https://github.com/mateimicu/kiss/blob/add_hotjar_support/exampleSite/config.toml#L53